app/src/test/java/com/example/beatbox/SoundTest.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.beatbox
+
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mock
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+
+@RunWith(MockitoJUnitRunner::class)
+class SoundTest {
+    private lateinit var subject: SoundViewModel
+
+    @Mock
+    private lateinit var sound: Sound
+
+    @Mock
+    private lateinit var beatBox: BeatBox
+
+    @Before
+    fun setUp() {
+        subject = SoundViewModel(beatBox)
+        subject.sound = sound
+    }
+
+    @Test
+    fun exposesSoundNameAsTitle() {
```

Comment:
This test should stay in the `SoundViewModelTest` class. Sorry for the confusion.

Here, in the `SoundTest` you could test the logic related to the `name` parameter `val name = assetPath.split("/").last().removeSuffix(WAV)` from the `Sound` class.

Basically you can create in instance of `Sound("string")` and verify that when `sound.name` is called it removes the ".wav" suffix